From 6982855f3c5f4772782e8fe27cf51666d4124b2c Mon Sep 17 00:00:00 2001
From: Jean-Philippe Lang <jp_lang@yahoo.fr>
Date: Sun, 25 Jun 2017 15:16:42 +0000
Subject: [PATCH 03/11] Unify query_form structure on calendar/gantt views
 (#26125).

This:
* moves the p with inline style on the calendar view (which
  wraps the navigation buttons) out of div#query_form_content
  and adds the contextual class to the element to unify this
  with the gantt view (and to remove the inline style from
  the element now that it's styled through a stylesheets'
  style [.contextual in fact]);
* restores the 12px font-size of the calendar navigation
  buttons;
* increases the font-size of the gantt zoom buttons to 12px;
* increases the margin of the calendar navigation and gantt
  zoom buttons to 12px 0px.

Patch by Mischa The Evil.

git-svn-id: http://svn.redmine.org/redmine/trunk@16703 e93f8b46-1217-0410-a6f0-8f06a7374b81
---
 app/views/calendars/show.html.erb  | 8 ++++----
 public/stylesheets/application.css | 1 +
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/app/views/calendars/show.html.erb b/app/views/calendars/show.html.erb
index e0ea4b0a6..060789b18 100644
--- a/app/views/calendars/show.html.erb
+++ b/app/views/calendars/show.html.erb
@@ -12,12 +12,12 @@
       <%= render :partial => 'queries/filters', :locals => {:query => @query} %>
     </div>
   </fieldset>
-
-  <p style="float:right;">
-    <%= link_to_previous_month(@year, @month, :accesskey => accesskey(:previous)) %> | <%= link_to_next_month(@year, @month, :accesskey => accesskey(:next)) %>
-  </p>
 </div>
 
+<p class="contextual">
+  <%= link_to_previous_month(@year, @month, :accesskey => accesskey(:previous)) %> | <%= link_to_next_month(@year, @month, :accesskey => accesskey(:next)) %>
+</p>
+
 <p class="buttons">
   <%= label_tag('month', l(:label_month)) %>
   <%= select_month(@month, :prefix => "month", :discard_type => true) %>
diff --git a/public/stylesheets/application.css b/public/stylesheets/application.css
index 16c4672c2..456a3c22b 100644
--- a/public/stylesheets/application.css
+++ b/public/stylesheets/application.css
@@ -364,6 +364,7 @@ div.table-list.boards .table-list-cell.name {width: 30%;}
 table.messages td.last_message {text-align:left;}
 
 #query_form_content {font-size:90%;}
+#query_form_with_buttons > p.contextual {font-size:12px; margin:12px 0px;}
 
 .query_sort_criteria_count {
   display: inline-block;
-- 
2.19.0.windows.1

